<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialog</class>
 <widget class="QDialog" name="FindReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="searchMode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Search Mode</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioNormalSearch">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Normal</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioExtendedSearch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>E&amp;xtended (\n, \r, \t, \0, \x...)</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>16</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radioRegexSearch">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Re&amp;gular expression</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxRegexMatchesNewline">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&amp;. matches newline</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="1" alignment="Qt::AlignRight">
      <widget class="QGroupBox" name="transparency">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Transparenc&amp;y</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioOnLosingFocus">
          <property name="text">
           <string>On losing focus</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioAlways">
          <property name="text">
           <string>Always</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="minimum">
           <number>20</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="sliderPosition">
           <number>70</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonFind">
         <property name="text">
          <string>Find</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCount">
         <property name="text">
          <string>Coun&amp;t</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonReplace">
         <property name="text">
          <string>&amp;Replace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonReplaceAll">
         <property name="text">
          <string>Replace &amp;All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonReplaceAllInDocuments">
         <property name="text">
          <string>Replace All in &amp;Opened Documents</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonFindAllInDocuments">
         <property name="text">
          <string>Find All in All &amp;Opened Documents</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonFindAllInCurrent">
         <property name="text">
          <string>Find All in Current Document</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonClose">
         <property name="text">
          <string>Close</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0,0,0,0">
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>8</number>
         </property>
         <property name="verticalSpacing">
          <number>8</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;Find what:</string>
           </property>
           <property name="buddy">
            <cstring>comboFind</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboFind">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="maxCount">
            <number>10</number>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelReplaceWith">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Replace with:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboReplace">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="maxCount">
            <number>10</number>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxBackwardsDirection">
         <property name="text">
          <string>Backward direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxMatchWholeWord">
         <property name="text">
          <string>Match &amp;whole word only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxMatchCase">
         <property name="text">
          <string>Match &amp;case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxWrapAround">
         <property name="text">
          <string>Wra&amp;p Around</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboFind</tabstop>
  <tabstop>comboReplace</tabstop>
  <tabstop>checkBoxBackwardsDirection</tabstop>
  <tabstop>checkBoxMatchWholeWord</tabstop>
  <tabstop>checkBoxMatchCase</tabstop>
  <tabstop>checkBoxWrapAround</tabstop>
  <tabstop>radioNormalSearch</tabstop>
  <tabstop>radioExtendedSearch</tabstop>
  <tabstop>radioRegexSearch</tabstop>
  <tabstop>checkBoxRegexMatchesNewline</tabstop>
  <tabstop>buttonFind</tabstop>
  <tabstop>buttonCount</tabstop>
  <tabstop>buttonReplace</tabstop>
  <tabstop>buttonReplaceAll</tabstop>
  <tabstop>buttonReplaceAllInDocuments</tabstop>
  <tabstop>buttonFindAllInDocuments</tabstop>
  <tabstop>buttonFindAllInCurrent</tabstop>
  <tabstop>buttonClose</tabstop>
  <tabstop>transparency</tabstop>
  <tabstop>radioOnLosingFocus</tabstop>
  <tabstop>radioAlways</tabstop>
  <tabstop>horizontalSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
